### PR TITLE
[ACTIONS/SOURCE] Aweber new actions and source new subscriber added

### PR DESCRIPTION
**Pull request by jcortes**
# Actions
- [x] Add Subscriber
- [x] Get Accounts
- [x] Get Lists
- [x] Get Subscribers   

# Source
- [x] Subscriber Added

### CODE REVIEW

**Comment by dylburger** [comment]
Thanks for submitting this PR! When we review PRs, we follow the [Pipedream component guidelines](https://pipedream.com/docs/components/guidelines/). If you're not familiar, here's a quick checklist:

- [ ] [Create components to address specific use cases whenever possible](https://pipedream.com/docs/components/guidelines/#component-scope)
- [ ] [Component `key`s should follow the format `app_name_slug-slugified-component-name`](https://pipedream.com/docs/components/guidelines/#component-key-pattern)
- [ ] Components should follow the [standard directory structure](https://pipedream.com/docs/components/guidelines/#folder-structure)
- [ ] [Prefer Node.js client libraries to REST APIs](https://pipedream.com/docs/components/guidelines/#using-apis-vs-client-libraries)
- [ ] When making API requests, [handle pagination](https://pipedream.com/docs/components/guidelines/#pagination) to ensure all data / events are processed
- [ ] [Use `secret` props](https://pipedream.com/docs/components/guidelines/#capturing-sensitive-data) to capture sensitive data
- [ ] Props and methods should be defined in [app files](https://pipedream.com/docs/components/guidelines/#app-files) whenever possible
- [ ] Document methods with [JS Docs](https://pipedream.com/docs/components/guidelines/#methods)
- [ ] [Use `optional` props](https://pipedream.com/docs/components/guidelines/#optional-vs-required-props) whenever possible, and [set a `default` value](https://pipedream.com/docs/components/guidelines/#default-values) where you can
- [ ] Use [async options](https://pipedream.com/docs/components/guidelines/#async-options) to accept user input wherever possible
- [ ] [Create subscription webhooks sources (vs polling sources) whenever possible](https://pipedream.com/docs/components/guidelines/#webhook-vs-polling-sources)
- [ ] [Always emit a `summary` with events](https://pipedream.com/docs/components/guidelines/#emit-a-summary)
- [ ] [Use a built-in deduping strategy wherever possible](https://pipedream.com/docs/components/guidelines/#deduping)
- [ ] When building webhook sources, [follow these best practices](https://pipedream.com/docs/components/guidelines/#webhook-sources)
- [ ] When building polling sources, [follow these best practices](https://pipedream.com/docs/components/guidelines/#polling-sources)

File: components/aweber/actions/add-subscriber/add-subscriber.mjs [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+      accountId,
+      listId,
+      data: {
+        email,
+        name,
+        tags,
+      },
+    });
+
+    $.export("$summary", `Successfully added subscriber with email ${email}.`);
```
Comment (michelle0927) [inline components/aweber/actions/add-subscriber/add-subscriber.mjs:66]: Does the Aweber API return anything when adding a subscriber? If so, we should return it here.
```suggestion
    const response = await this.aweberApp.addSubscriber({
      $,
      accountId,
      listId,
      data: {
        email,
        name,
        tags,
      },
    });

    $.export("$summary", `Successfully added subscriber with email ${email}.`);
    
    return response;
```

**Review by michelle0927 — CHANGES_REQUESTED** [review]

File: components/aweber/actions/add-subscriber/add-subscriber.mjs (hunk 1 above)
Comment (jcortes) [reply components/aweber/actions/add-subscriber/add-subscriber.mjs:66]: No it doesn't return anything here, that's why I used `email` prop in the summary

**Review by michelle0927 — APPROVED** [review]

**Comment by DilanAthukorala** [comment]
@jcortes I am not getting events for new_subscriber source after adding new subscribers and when they are getting subscribed.

**Comment by DilanAthukorala** [comment]
@jcortes I'm moving this back to changes required since there are no events triggered for this after subscribing new users. Please let me know if there's any specific steps needs to be done from Aweber, so we can add that to the documentation.

**Comment by DilanAthukorala** [comment]
Verification completed.

**Review by michelle0927 — APPROVED** [review]

**Pull request merged**